### PR TITLE
Rename: StatusCode -> Code

### PR DESCRIPTION
**Pull request by izumin5210**
`StatusCode` is too long...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,8 +249,8 @@ func ReportError(c *gin.Context, err error) {
 	}
 
 	// Set status code accordingly
-	if appErr.StatusCode > 0 {
-		c.Status(appErr.StatusCode)
+	if appErr.Code > 0 {
```
Comment (creasty) [inline README.md]: This example is no longer valid...

File: options.go [hunk 2]
```diff
@@ -10,10 +10,10 @@ func WithMessage(msg string) Option {
 	}
 }
 
-// WithStatusCode annotates with the status code.
-func WithStatusCode(code interface{}) Option {
+// WithCode annotates with the status code.
```
Comment (creasty) [inline options.go]: `with the status code` → `with the error code.`

File: README.md [hunk 3]
```diff
@@ -144,8 +144,8 @@ type Error struct {
 	Err error
 	// Message is an annotated description of the error
 	Message string
-	// StatusCode is a status code that is desired to be used for a HTTP response
-	StatusCode int
+	// Code is a status code that is desired to be used for a HTTP response
+	Code int
```
[the remark this answers is not in the document]
Comment (izumin5210) [reply README.md]: 🙈

**Comment by izumin5210** [comment]
fixed 🏥

**Pull request merged**